site/content/en/docs/administration/advanced/k8s_deployment_with_helm.md

```diff
@@ -319,3 +326,26 @@ cvat:
             claimName: my-claim-name
 
 ```
+### Im receiving service unavailable errors
+It is possible that the migrations did not run automatically, you may need to run them manually:
+```sh
+HELM_RELEASE_NAMESPACE="<desired_namespace>" &&\
+HELM_RELEASE_NAME="<release_name>" &&\
+BACKEND_POD_NAME=$(kubectl get pod --namespace $HELM_RELEASE_NAMESPACE -l tier=backend,app.kubernetes.io/instance=$HELM_RELEASE_NAME -o jsonpath='{.items[0].metadata.name}') &&\
+kubectl exec -it --namespace $HELM_RELEASE_NAMESPACE $BACKEND_POD_NAME -c cvat-app-backend-server-container -- python manage.py migrate &&\
+kubectl exec -it --namespace $HELM_RELEASE_NAMESPACE $BACKEND_POD_NAME -c cvat-app-backend-server-container -- python manage.py health_check
+```
+### Im getting multi attach volume errors when my backend pods run on different nodes
```

Comment:
There is new option in values.yaml for it now

Reply:
You mean to allow ReadWriteMany via the values.yaml? I think thats a great addition, and i even pulled #6137 while it was still open into my deployment. but in my case our volumes dont support ReadWriteMany, so I (and i imagine many other people) would still need to use ReadWriteOnce with podAffinity.